libs/core/src/lib/multi-input/multi-input.component.html

```diff
@@ -83,9 +87,13 @@
                 [attr.aria-required]="required"
                 [class.fd-input--compact]="compact"
                 (ngModelChange)="_handleSearchTermChange($event)"
+                [attr.aria-haspopup]="!mobile"
+                [attr.aria-expanded]="!!(open && !mobile && displayedValues && displayedValues.length)"
                 (keydown)="_handleInputKeydown($event)"
                 [ngModelOptions]="{ standalone: true }"
                 [attr.id]="inputId"
+                [attr.aria-describedby]="tokenHiddenId"
+                [attr.aria-roledescription]="roleDescription"
```

Comment:
Do we really need `describedby` and `roledescription` at the same time?
From the [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description#description) it says `describedby` has precedence in this case which means `roledescription` will be ignored

Reply:
<img width="375" alt="Screenshot 2021-11-23 at 9 28 51 AM" src="https://user-images.githubusercontent.com/53534379/142969610-d4d24017-0b60-4576-9d9e-a1d4919259a5.png">

it reads both in jaws

Reply:
ok